chess/3-web-api/web-api.md

```diff
@@ -417,8 +417,9 @@ After your code has successfully been auto-graded, a TA will review the code in
 | Category       | Criteria                                                                                                                                                                                         |       Points |
 | :------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -----------: |
 | GitHub History | At least 10 GitHub commits evenly spread over the assignment period that demonstrate proof of work                                                                                               | Prerequisite |
-| Web API Works  | All pass off test cases in `StandardAPITests.java` succeed                                                                                                                                       |          115 |
-| Web Page Loads | Test web page properly loads in browser (including all files linked to by the test page: favicon.ico, index.css, index.js)                                                                       |           10 |
+| Web Page Loads | Test web page properly loads in browser (including all files linked to by the test page: favicon.ico, index.css, index.js)                                                                       | Prerequisite |
```

Comment:
Do we really need this? I assume the point of removing the 10 point award specifically for the web page is due to the new test that makes sure it loads, and therefore is included in the 125 points for the tests. We don't call out any other functionality from the tests as a specific prerequisite. Why this one?